tasks/libs/github_actions_tools.py

```diff
@@ -94,8 +94,8 @@ def trigger_macos_workflow(
                         return recent_run
             else:
                 print("waiting for jobs to popup...")
-                sleep(3)
-        sleep(5)
+                sleep(5)
+        sleep(10)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The `10` should be define next to the `MAX_RETRIES` IMO, else the `Retry for up to 5 minutes` can be a little confusing